In Jhipster I deleted filed of entity through command and after run “ gradlew liquibaseDiffChangelog ” command but got this error

FAILURE: Build failed with an exception.

* Where:
Script 'G:\Project\beequote\gradle\liquibase.gradle' line: 19

* What went wrong:
Execution failed for task ':pathingLiquibaseJar'.
> Could not resolve all files for configuration ':liquibase'.
   > Could not find org.liquibase.ext:liquibase-hibernate5:.
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more l
og output.

* Get more help at https://help.gradle.org

BUILD FAILED in 24s

--- a/generators/server/templates/_gradle.properties
+++ b/generators/server/templates/_gradle.properties
@@ -34,6 +34,8 @@ spring_boot_version=1.5.9.RELEASE
 hibernate_version=5.2.12.Final
 mapstruct_version=1.2.0.Final
 
+liquibase_hibernate5_version=3.6
+
 ## below are some of the gradle performance improvement settings that can be used as required, these are not enabled by default
 
 ## The Gradle daemon aims to improve the startup and execution time of Gradle.

--- a/generators/server/templates/gradle/_liquibase.gradle
+++ b/generators/server/templates/gradle/_liquibase.gradle
@@ -23,7 +23,7 @@ configurations {
 }
 
 dependencies {
-    liquibase "org.liquibase.ext:liquibase-hibernate5"
+    liquibase "org.liquibase.ext:liquibase-hibernate5:${liquibase_hibernate5_version}"
 }
 
 if (OperatingSystem.current().isWindows()) {

